src/Jss.js

```diff
@@ -15,6 +15,11 @@ export default class Jss {
     this.sheets = new SheetsRegistry()
     this.plugins = new PluginsRegistry()
     this.uid = uid
+    if (__VERSION__) {
```

Comment:
Yeah I searched babel plugins to do it before creating patch. To keep not adding more build tools, I didn't use it because `package.json` can be ugly:
```
    "build:lib": "V=`npm run v -s` && __VERSION__=$V babel src --out-dir lib",
    "v": "node -e \"console.log(require('./package.json').version)\""
```

Also, `src` files are called from `test` codes so we need to add `require('../package.json').version`, but if we don't add this line, `dist/jss.js` and `dist/jss.min.js` actually contain `package.json` data like this: https://gist.github.com/watilde/7aacb5e924fc7317308dbab367a37d89#file-jss-js-L1602-L1668